example-app/SantokuApp/.eslintrc.js

```diff
@@ -124,6 +129,15 @@ module.exports = {
     ],
     // https://eslint.org/docs/latest/rules/curly
     curly: ['error', 'all'],
+    // https://eslint.org/docs/latest/rules/no-restricted-syntax
+    'no-restricted-syntax': [
+      'error',
+      {
+        selector: 'ForInStatement',
+        message:
+          'for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array.',
```

Comment:
> `Use Object.{keys,values,entries}`

array の for in の場合も同じメッセージだと混乱しそうな気がします。

object の for inだけに絞れるのがベストですが、少し調べた感じやり方が見つからなかった、メッセージを調整でしょうか

Reply:
こんな感じですかね？
```suggestion
          'for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use for...of or Array.{forEach,map} instead. For Object, use Object.{keys,values,entries}, and iterate over the resulting array.',
```

Reply:
こんなのでどうでしょう？
- `For Object` と対応する `For Array` がほしい
- 厳密には `forEach`, `map` は `Array` のmethodじゃない (`Object.{keys,values,entries}` との対比)
- `for of` は優先度低いので後にしたい
```suggestion
          'for..in loops iterate over the entire prototype chain, which is virtually never what you want. For Array, use array.{forEach,map} instead. For Object, use Object.{keys,values,entries}, and iterate over the resulting array. If it is difficult to describe in method, use for...of.',
```
（最後の1文は自身無いです）